docs/user-guide/creating-application/deployment-template/rollout-deployment.md

```diff
@@ -249,19 +259,22 @@ ingressInternal:
       - host: example1.com
         paths:
             - /example
+        pathType:
       - host: example2.com
         paths:
             - /example2
             - /example2/healthz
+        pathType:
   tls: []
 ```
 
 | Key | Description |
 | :--- | :--- |
 | `enabled` | Enable or disable ingress |
 | `annotations` | To configure some options depending on the Ingress controller |
-| `path` | Path name |
 | `host` | Host name |
+| `path` | Path name |
+| `pathType` | Path type. Supported path types are `ImplementationSpecific`, `Exact` and `Prefix`.|
```

Comment:
Only write, "Supported path types are `ImplementationSpecific`, Exact` and `Prefix`

Reply:
done